### ISSUE TITLE
Transmission: existing + no new transmission

### ISSUE THREAD

**Issue by holgerteichgraeber** [YoungFaithful/CapacityExpansion.jl#30]
@YoungFaithful 
I would like to run a capacity expansion model with existing transmission, but not allow any newly build transmission. The documentation suggests the following (thanks for putting that in there!):
> Include transmission=true and infrastructure = Dict{String,Array}("existing"=>[...,"transmission"], "limit"=>[...,"transmission"]) to model existing transmission and limit the total transmission TRANS to the values defined in the lines.csv file. If no new transmission should be setup, use the same values for existing transmission and the limit.

However, the following code seems to build new transmission beyond what is defined in `lines.csv` as `power_ex`:

```julia
region = "CA_14"
ts_input_data = load_timeseries_data_provided(region)
cep_data = load_cep_data_provided(region)
optimizer=Clp.Optimizer
ts_clust_result = run_clust(ts_input_data; method="kmeans", representation="centroid", n_init=5, n_clust=5)
ts_clust_data = ts_clust_result.clust_data
test_result = run_opt(ts_clust_data,cep_data,optimizer;descriptor="Test",transmission = true,infrastructure = Dict{String,Array}("existing"=>["demand","transmission"], "limit"=>["transmission"]),  limit_emission=Dict{String,Number}("CO2/electricity"=>1000));
test_result.variables["TRANS"]
```

Two questions:
1) What am I missing when setting up the problem?
2) I see that CA14 data does not have exisiting transmission defined (all 0). Do we have that data somewhere? Would love to play with that.

**Comment by YoungFaithful** [comment]
did you assign a power_lim with the same value as power_ex?

**Comment by holgerteichgraeber** [comment]
No, I didn't. I see, in `lines.csv`. Thanks!

**Comment by YoungFaithful** [comment]
Merged #31
